internal/pkg_npm/pkg_npm.bzl

```diff
@@ -109,6 +118,12 @@ You can use values from the workspace status command using curly braces, for exa
 See the section on stamping in the [README](stamping)
 """,
     ),
+    "tgz": attr.string(
```

Comment:
can it be an `attr.output` ?

Reply:
oh wait I see, this attribute isn't actually used? this is just here to document how the macro works? smelly

Reply:
👃 Yeah, perhaps we refactor it like you suggested to be like ts_project macro, for another PR as that seems an independent change